src/printer/tvmscript_unified_printer.cc

```diff
@@ -0,0 +1,498 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include "tvmscript_unified_printer.h"
+
+#include <tvm/node/functor.h>
+
+#include <cstdint>
+#include <string>
+
+#include "doc.h"
+#include "tvm/runtime/container/array.h"
+#include "tvm/runtime/container/string.h"
+#include "tvm/runtime/object.h"
+#include "tvm/tir/expr.h"
+#include "tvm/tir/function.h"
+#include "tvm/tir/op.h"
+#include "tvm/tir/stmt.h"
+
+namespace tvm {
+
+namespace {
+using namespace tir;
+using runtime::ArrayNode;
+}  // namespace
```

Comment:
no need for anonymous namespace in the header. also, generally speaking we don't use "using namespace" in headers

Reply:
Removed the anonymous namespace. I keep the `using namespace` just for now. It will go eventually as we move those doc producer definition to the tir namespace